### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Aladin Iseni, Arthur Jacobs

### DISCUSSION THREAD

**Discussion by Arthur2479** [heig-vd-dai-course/heig-vd-dai-course#467] [Show and tell]
### Welcome to:

![](https://img.shields.io/badge/No--tion-black?style=for-the-badge)

→ Notion, but much, much worse.

 A simple CLI tool (or with a graphical interface if we feel audacious) to interact with a server, storing text notes.
 Some functionalities we'd like to implement if we have time:
- Allow to create lists, which are like notes but the user can just send a new string that will be appended to the list
- Apply a Caesar code to the note you query (assume the note was sent encoded)
- A super useful functionality: the notes you will take will be saved approximately (for example by switching the letters inside them, see [Typoglycemia](https://en.wikipedia.org/wiki/Transposed_letter_effect#:~:text=their%20separate%20parts.-,Internet%20meme,-%5Bedit%5D))

Organisation link (for this, previous and future practical works): https://github.com/HEIG-VD-DAI-Iseni-Jacobs

**Comment by Arthur2479** [comment]
@aladin-heig

**Comment by GeraudSilvestri** [comment]
It's good for us.

**Comment by Arthur2479** [comment]
Here is the commit hash:
ab300f26d34ba8119d0be65690f6be55fe057429
And the link to the repo:
https://github.com/HEIG-VD-DAI-Iseni-Jacobs/no-tion

@aladin-heig
